Formula/z/zrok.rb

```diff
@@ -3,8 +3,8 @@
 class Zrok < Formula
   desc "Geo-scale, next-generation sharing platform built on top of OpenZiti"
   homepage "https://zrok.io"
-  url "https://github.com/openziti/zrok/archive/refs/tags/v0.4.34.tar.gz"
-  sha256 "7ba4c46809f39876723cf4647e9665c78df0b4382c3b629db848d07d803862a8"
+  url "https://github.com/openziti/zrok/archive/refs/tags/v0.4.35.tar.gz"
+  sha256 "02a8ac4a44882d52cadcdd831827d7338ae53e370f6f19e09ff19baf2f3e6719"
```

Comment:
```suggestion
  sha256 "89a2160e3f2584e6ad1bf6f28bb4e8d43fb498ab783c12332594a16bb3a43b9a"
```

@ziti-ci that does not look good